src/read/line.rs

```diff
@@ -413,24 +373,87 @@ fn yaw_field<'a>(
                     }
 
                     let (i, count) = non_zero_u32(i)?;
-                    Ok((
-                        i,
-                        Some(AutoMovement::Strafe(StrafeSettings {
-                            type_,
-                            dir: StrafeDir::RightLeft(count),
-                        })),
-                    ))
+                    (i, StrafeDir::RightLeft(count))
                 }
+                // For Left or Right
                 dir => {
-                    // Skip the field if constant yawspeed and side strafing.
+                    // ConstYawspeed uses yaw field for Left or Right.
+                    // Therefore, we cannot parse it with tag("-").
                     let (i, _) = if is_constant_yawspeed {
-                        recognize_float(i)?
+                        tag("")(i)?
```

Comment:
Does this line do anything?

Reply:
I think it is just a nice that it shows this line explicitly does nothing.